scripts/get-vm-ip.py

```
@@ -0,0 +1,209 @@
+
+import argparse
+import os
+import subprocess
+import sys
+import time
+from typing import Optional
+
+
+def _run(cmd: list[str]) -> tuple[int, str, str]:
+    """Run a command and return (returncode, stdout, stderr)."""
+    result = subprocess.run(cmd, capture_output=True, text=True)
```

Ruff (0.14.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)